pkg/meta/base.go

```diff
@@ -295,7 +296,18 @@ func (m *baseMeta) calcDirStat(ctx Context, ino Ino) (space, inodes uint64, err
 }
 
 func (m *baseMeta) GetDirStat(ctx Context, inode Ino) (space, inodes uint64, err error) {
-	return m.en.doGetDirStat(ctx, m.checkRoot(inode))
+	var trySync bool
```

Comment:
this variable is not necessary, use `!m.conf.ReadOnly` directly

Reply:
Removed